doc/project/checklist.md

```diff
@@ -343,7 +353,7 @@ All assumptions are implemented and reviewed.
 
 ### SIM_FW_SIMULATED
 
-For chip-level, verified pieces of FW code (DIFs) in simulaton.
+For chip-level, verified pieces of FW code in simulaton.
```

Comment:
I'm not sure the change is needed but, if you're touching this line anyway, there's a missing i in simulat**i**on.

Reply:
Isn't DV now meant to be the only user of DIFs? If so, this sentence should stay as it was.